### ISSUE TITLE
텍스트파일이 존재하지 않는 경우 예외처리 질문드립니다.

### ISSUE THREAD

**Issue by aliomigna** [DSLDataStorage/DS_Project_2_2020_2#16]
제안서 Requirements in implenmentation의 마지막에
<읽어야 할 텍스트 파일이 존재하지 않으면 해당 텍스트 파일을 생성한 뒤 진행하도록 한다>로 제시되었는데,
Functional Requirements의 요구사항에서는 
<텍스트파일이 존재하지 않는 경우 에러 코드를 출력>하는 것으로 제시되어있습니다.

프로그램에서 읽어서 사용하는 txt파일인 (market.txt, result.txt)와 (command.txt)의 예외처리 방법이 서로 달라서
 market.txt, result.txt가 없는 경우 에러코드를 출력하고,
command.txt가 없는 경우 command.txt를 새로 생성하고, 생성한 빈 명령어 파일을 읽어 동작하면 되는 것 인가요?

**Comment by DSLDataStorage** [comment]
예 맞습니다. 해당 문구는 command.txt에 대한 문구입니다. 그부분에 대해 이해하기 쉽도록 문서를 수정하여 다시 업로드 해드리겠습니다.
